oauth2_provider/settings.py

```diff
@@ -54,6 +54,7 @@
     "ID_TOKEN_EXPIRE_SECONDS": 36000,
     "REFRESH_TOKEN_EXPIRE_SECONDS": None,
     "REFRESH_TOKEN_GRACE_PERIOD_SECONDS": 0,
+    "REFRESH_TOKEN_REUSE_PROTECTION": False,
```

Comment:
I've set the default to False for compatibility reasons (if someone has a faulty client that due to a bug reuses the token, it might cause accidental logouts), but at the same time you could also argue it is better to have the most secure option as default.